src/ts/state/store.ts

```diff
@@ -100,4 +100,6 @@ class Store<
   }
 }
 
+export type IStore<TMatch extends IMatch = IMatch> = Store<TMatch>;
```

Comment:
Should this be done in a new file in the interface folder, or would that be overkill?

Reply:
It's a great point. We're quite inconsistent in this project – sometimes interfaces are defined inline, sometimes they have their own files. One reason is that if an interface isn't clearly 'owned' by a piece of code (e.g. match and block interfaces), it's difficult to know which code to colocate it with. For other interfaces (e.g. `IStore` above) it's clearer where to put things.

It'd be nice to establish a convention, document it, refactor where necessary and stick with it. That change probably wants its own PR.